rxandroidble/src/main/java/com/polidea/rxandroidble/internal/operations/RxBleRadioOperationMtuRequest.java

```diff
@@ -0,0 +1,65 @@
+package com.polidea.rxandroidble.internal.operations;
+
+import android.bluetooth.BluetoothGatt;
+import android.os.Build;
+import android.support.annotation.NonNull;
```

Comment:
This import is unused according to https://travis-ci.org/Polidea/RxAndroidBle/builds/181702296 checkstyle.